subsys/bluetooth/audio/capabilities.c

```diff
@@ -28,6 +28,9 @@
 static sys_slist_t snks;
 static sys_slist_t srcs;
 
+IF_ENABLED(CONFIG_BT_PAC_SNK, (static enum bt_audio_context sink_available_contexts;))
```

Comment:
Was actually not aware of the `IF_ENABLED` macro until now (this will also be the first use of it in the Bluetooth subsystem). 

Is it recommended to use this instead of `#if defined(...)`? 

It doesn't really look like it can be used with with an `else`, and personally I don't think it helps readability at all. I'm not going to block the PR based on this, but just giving my personal input :)

Reply:
I personally, don't like the code chopped by `#ifdef`'s. With lot of them the code looks messy

Reply:
It's a personal preference :D